modules/permissions/common_permissions.tf

```diff
@@ -122,4 +122,20 @@ resource "aws_iam_policy" "custodian_get_parameters" {
   name        = "TDRCustodianGetParameters"
   description = "Policy to allow Cloud Custodian to get SSM parameters"
   policy      = data.aws_iam_policy_document.custodian_get_parameters.json
-}
\ No newline at end of file
+}
+
+
+resource "aws_iam_role" "jenkins_lambda_deploy_role" {
+  name = "TDRJenkinsNodeLambdaRoleMgmt"
```

Comment:
Should environment "Mgmt" be hardcoded?

Reply:
Technically it's fine but it is a bit rubbish. I've added a variable in and passed it in hard coded from the main root.tf file like the other modules.